packages/gatsby/src/bootstrap/resolve-module-exports.js

```diff
@@ -118,3 +109,25 @@ https://gatsby.dev/no-mixed-modules
   }
   return exportNames
 }
+
+/**
+ * Given a `require.resolve()` compatible path pointing to a JS module,
+ * return an array listing the names of the module's exports.
+ *
+ * Returns [] for invalid paths and modules without exports.
+ *
+ * @param {string} modulePath
+ * @param {string} mode
+ * @param {function} resolver
+ */
+module.exports = (modulePath, { mode = `analisys`, resolver } = {}) => {
```

Comment:
```suggestion
module.exports = (modulePath, { mode = `analysis`, resolver } = {}) => {
```

Reply:
This doesn't _really_ matter here--but for correctness 🤷‍♂️

Reply:
lol 🙈 I want autocorrect in vs code 😂